src/routes/NewUser/components/NewUserPage/NewUserPage.js

```diff
@@ -63,7 +64,7 @@ function NewUser() {
       try {
         const ref = firestore.doc(`${USERS_COLLECTION}/${user.uid}`);
         // Call it once because this will throw the permission exception.
-        await ref.get();
+        setUserData(await ref.get());
```

Comment:
Can we not do the single get and use the data in both state updates? Or is that what you are trying to say in the comment right above? Since the first one errors?